data/entities.yaml

```diff
@@ -137,6 +137,50 @@
     water or steam.
   properties: [portable]
 
+- name: iron plate
+  display:
+    attr: iron
+    char: '■'
+  description:
+  - Raw iron ore. Used to create more resilient tools.
+  - It can only be mined by drilling in the mountains.
```

Comment:
Seems like this description got copied from `iron ore` and never edited?

Reply:
Fixed :+1: